polymerfire.html

```diff
@@ -10,4 +10,7 @@
 <link rel="import" href="firebase-document.html">
 <link rel="import" href="firebase-query.html">
 <link rel="import" href="firebase-messaging.html">
-<link rel="import" href="firebase-storage-script.html">
```

Comment:
because firebase-storage-multiupload, firebase-storage-upload-task and firebase-storage-ref already include it